auslib/test/admin/views/test_releases.py

```diff
@@ -1729,6 +1765,71 @@ def testGetRevisions(self):
         with self.assertRaises(KeyError):
             data['data']
 
+    def testGetHistory(self):
+        url = '/releases/history'
+        ret = self._get(url)
+        self.assertEquals(ret.status_code, 200, msg=ret.data)
+        data = json.loads(ret.data)
+        self.assertEquals(data["count"], 3)
+        expected = {
+            'count': 3,
+            'revisions': [
+                {
+                    'product': 'b', 'data_version': 1, 'changed_by': 'bill', '_different': [], 'read_only': 'False',
+                    'change_id': 6, '_time_ago': '48 years and 583 months ago', 'name': 'b', 'timestamp': 16
```

Comment:
I don't think these _time_ago fields are continue be accurate forever. It looks like they'll be wrong starting in ~1 month. You're not actually comparing them below as far as I can tell, so you may as well just remove them here.

Reply:
Ok, I'll take a closer look at that and fix as necessary.